src/main/java/frc/robot/utilities/LimelightHelpers.java

```diff
@@ -432,15 +435,37 @@ public static URL getLimelightURLString(String tableName, String request) {
   }
   /////
   /////
+  private static double horizAngle;
+  private static double vertAngle;
 
   public static double getTX(String limelightName) {
+    if (Robot.isSimulation()) {
+      return horizAngle;
+    }
     return getLimelightNTDouble(limelightName, "tx");
   }
 
   public static double getTY(String limelightName) {
+    if (Robot.isSimulation()) {
+      return vertAngle;
+    }
     return getLimelightNTDouble(limelightName, "ty");
   }
 
+  public static void simPose(SwerveDriveState state) {
+    double newX =
+        state.Pose.getX()
+            - (SmartDashboard.getBoolean("IsRedAlliance", true) == true ? (8.308467 * 2) : 0);
+    double newY = state.Pose.getY() - (1.451102 + 3.837865);
+    double horizontalAngle = Math.atan(newY / newX);
+    double verticalAngle = Math.atan(0.4572 / Math.sqrt(Math.pow(newX, 2) + Math.pow(newY, 2)));
+    horizAngle = horizontalAngle;
+    vertAngle = verticalAngle;
+    SmartDashboard.putNumber("Horizontal Angle", horizAngle);
+    SmartDashboard.putNumber("Vertical Angle", vertAngle);
+    SmartDashboard.putNumber("X Distance", newX);
```

Comment:
Adding 'LL' prefix would be nice to distinguish what these are displaying when looking at SmartDashboard